database/models/room.js

```diff
@@ -20,6 +21,7 @@ const findAll = (callback) => {
 };
 
 const findByID = (id, callback) => {
+  console.log('were in find by id and this is id', id);
```

Comment:
Consider removing this log unless it something we're not worried about the user seeing.
Otherwise, everything looks solid dude.

Reply:
Thank you! You are absolutely right + good catch, going to remove it now.